ffi/rust/firedancer-sys/src/tango/mod.rs

```diff
@@ -3,13 +3,17 @@ mod dcache;
 mod fctl;
 mod fseq;
 mod mcache;
+mod mvcc;
 mod tcache;
+mod validator;
 mod xdp;
 
 pub use cnc::*;
 pub use dcache::*;
 pub use fctl::*;
 pub use fseq::*;
 pub use mcache::*;
+pub use mvcc::*;
 pub use tcache::*;
+pub use validator::*;
```

Comment:
Not sure about "validator", maybe something about leader schedule?

Reply:
I'm not sure about it either - I was thinking it might be good to have a directory for all the solana-labs FFI, e.g. leader schedule, staking, slot timing, etc.

I'll change it and we can revisit once we have the other stuff on main.